e2e-tests/Cargo.toml

```diff
@@ -1,12 +1,12 @@
 [package]
 name = "aleph-e2e-client"
 version = "0.1.0"
-edition = "2018"
+edition = "2021"
```

Comment:
there is also a bunch of rustfmt files that shoudl be bumped as well

Reply:
I bumped edition in two corresponding files